src/Hazel.hs

```diff
@@ -180,6 +183,8 @@ data Type
 data Usage = Inexhaustible | UseOnce | Exhausted
```

Comment:
So is exhausted like irrelevant ? Or does it only apply to bindings that are use once?